api/applicationserver.proto

```diff
@@ -151,3 +151,32 @@ service AsEndDeviceRegistry {
   };
 }
 
+// MQTTConfig represents the configuration of the Application Server MQTT frontend.
+message MQTTConfig {
+  // The public listen address of the frontend.
+  string public = 1 [(validate.rules).string.uri = true];
```

Comment:
No need to validate response messages, but ok.

Reply:
I disagree:
1. It's extremely useful when e.g. storing messages to registries, since we use fieldmasks. Missing a `public` fieldpath here would instantly fail the transaction and would be fixed, however otherwise it could easily go unnoticed and cause hard-to-debug issues/panics in production.
2. It's extra context/documentation.

Reply:
I don't think it hurts either.

Consider `public_address` here for the name